troubadix/standalone_plugins/changed_cves.py

```diff
@@ -0,0 +1,98 @@
+import re
+from argparse import ArgumentParser, Namespace
+from subprocess import CalledProcessError
+
+from pontos.terminal.terminal import ConsoleTerminal
+
+from troubadix.argparser import file_type
+from troubadix.helper.patterns import (
+    SpecialScriptTag,
+    get_special_script_tag_pattern,
+)
+from troubadix.standalone_plugins.common import get_merge_base, git
+
+CVE_PATTERN = re.compile(r"CVE-\d{4}-\d{4,7}")
```

Comment:
```suggestion
CVE_PATTERN = re.compile(r"CVE-\d{4}-\d{4,}")
```

Technically its "4 or more digits", practically it is probably irrelevant

Reply:
You're indeed right. I could have sworn that there was a limitation, but there indeed isn't.